src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -18,11 +18,31 @@
         {% endif %}
     {% endset %}
 
+    {% if collector.forward %}
```

Comment:
In the same template. We already do that in other panels. Example: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig